spec/kitchen_spec/directions/bake_toc_spec.rb

```diff
@@ -301,16 +302,16 @@
     end
   end
 
-  let(:preface_page) do
-    page_element(
+  let(:no_pagetype_page) do
+    book_containing(html:
       <<~HTML
-        <div data-type="page" id="p1" class="preface">
+        <div data-type="page" id="p1" data-sm="/module/m240/filename.cnxml:8:22">
           <h1 data-type="document-title">
             <span data-type="" itemprop="" class="os-text">Preface</span>
           </h1>
         </div>
       HTML
-    )
+    ).pages.first
```

Comment:
So for some reason, the original test for this was a page with a page inside it? And it was running the test on the outer (hidden) page???